oscplot.c

```diff
@@ -7030,8 +7018,18 @@ static void create_plot(OscPlot *plot)
 	gtk_databox_create_box_with_scrollbars_and_rulers(&priv->databox, &table,
 		TRUE, TRUE, TRUE, TRUE);
 	gtk_box_pack_start(GTK_BOX(priv->capture_graph), table, TRUE, TRUE, 0);
-	// TO DO: handle this using GTK3 (probably using stylesheets)
-	// gtk_widget_modify_bg(priv->databox, GTK_STATE_NORMAL, &color_background);
+
+	GtkCssProvider *provider;
+	GtkStyleContext *style_context;
+	GdkDisplay *display;
+	GdkScreen *screen;
+	provider = gtk_css_provider_new();
```

Comment:
Should the 'provider' be freed at some point?

Reply:
provider is now freed when plot object is destroyed